Formula/quickemu.rb

```diff
@@ -0,0 +1,37 @@
+class Quickemu < Formula
+  desc "Quickly create and run optimised Windows, macOS and Linux virtual machines"
+  homepage "https://github.com/quickemu-project/quickemu"
+  url "https://github.com/quickemu-project/quickemu/archive/refs/tags/4.9.7.tar.gz"
```

Comment:
Can this be made 'dynamic' to point to the latest release tag or do we need to extend the release generating CI to update the old release tag to the "new" one when a release is tagged and prepared?